kubectl_plugins/create-service-broker/create-service-broker.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/kubernetes-incubator/service-catalog/kubectl_plugins/utils"
+	v1alpha1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	clientset "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+
+	"k8s.io/client-go/rest"
+)
+
+const USAGE = `
+
+INVALID USAGE:
+
+Usage:
+  kubectl plugin create-service-broker BROKER_NAME BROKER_URL
```

Comment:
For a follow-up: We should plumb through auth somehow.  We should assume basic auth will be one of many options.  Basic auth is stored in a secret, so we will need to plumb through the namespace and name of the secret.